js/blog.js

```diff
@@ -0,0 +1,25 @@
+const subscribe = () => {
+
+    var subscribeForm = document.querySelector('#subForm');
+    subscribeForm.addEventListener('submit', (e) => {
+        e.preventDefault();
+
+        db.collection('subscribes').add({
```

Comment:
If it was me, I would call this collection subscriptions. Subscribes is not a name.